release/ray_release/byod/byod.Dockerfile

```diff
@@ -15,8 +15,13 @@ sudo apt-get update -y \
     && sudo apt-get install -y --no-install-recommends $(cat requirements_debian_byod.txt) \
     && sudo apt-get autoclean
 
+rm -rf /tmp/wrk
+git clone --branch 4.2.0 https://github.com/wg/wrk.git /tmp/wrk
+make -C /tmp/wrk -j
```

Comment:
optional: we can probably also just save a copy of the `wrk` in our s3 or something.

Reply:
or maybe even remove it as a dependency, it probably will break some day ;)